### DISCUSSION TITLE
What are the conditions to become a contributor?

### DISCUSSION THREAD

**Discussion by cassiofb-dev** [HyunCafe/contribute-practice#26] [Q&A]
Hi, I made a pull request here 2 weeks ago. I noticed that even though it was merged my name wasn't listed on contributors, so I decided to wait some time, but even now my name doesn't show up there.

- My pull request: #21 
- My commits:
  - 873549372567062741825fb1e9d2092382cda9c8
  - 8739c277f7110a473ddbcf64b80b5cb386adbe4b
  - 873549372567062741825fb1e9d2092382cda9c8
- [The contributors](https://github.com/HyunCafe/contribute-practice/graphs/contributors) (I'm not here 🥹)

**Comment by HyunCafe** [comment]
oh weird, you should be listed, im not sure what makes a condition to be listed ( I think github sets that) - you can try editing the contribution again perhaps? or what can I do to add you on I am not sure haha

**Comment by HyunCafe** [comment]
maybe did you make the contribution through github editing it? I think you
need to create your own branch and merge maybe,

**Comment by cassiofb-dev** [comment]
@HyunCafe I found the problem, I can't be a contributor if I edited a personal branch T-T, I'm so sad my beautiful face will never appear lol.

The problem is that I created my own branch, github says the following:
> Commits will appear on your contributions graph if they meet all of the following conditions: The commits were made: In the repository's default branch

![image](https://user-images.githubusercontent.com/61878987/221909147-8f07350a-eb71-4fb9-8b1b-662e22e30668.png)

**Reply by cassiofb-dev** [reply]
It's quite a shame because I think it's a good practice to group commits in another branch and them merge.
I'm marking the reply above as answer T-T

**Reply by HyunCafe** [reply]
Hmm that seems to be for commit contributionson the graph (on your personal GitHub profile) to be counted not for contributions on an open source, if your forked my project and made a commit and merged it should work, if not you can try editing the readme directly from GitHub only my repo and see if that triggers the condition, worth a shot

**Reply by cassiofb-dev** [reply]
Yeah, I'll see it deeper by checking what I did different from people that actually shows up.
